rebar.config

```diff
@@ -20,12 +22,12 @@
        ]}.
 
 {relx, [{release, {aerepl, "2.0.0"},
-         [aerepl, {aechannel, load}, {aecore, load}, {aeutils, load}]},
-        {lib_dirs, ["node/_build/dev1/rel/aeternity/lib"]},
+         [aerepl, {aechannel, load}, {aecore, load}, {aeutils, load}, {mnesia, load}, sasl]},
+        {lib_dirs, ["node/_build/dev1/rel/aeternity/lib", "_build/prod/lib/"]},
         {dev_mode, true},
         {vm_args, "config/vm.args"},
         {sys_config, "config/sys.config"},
-        {include_erts, false},
+        {include_erts, true},
```

Comment:
Why do you need to include erts?

Reply:
I don't remember, it was a fix to something but I forgot

Reply:
I don't think this is really necessary, since erts is already included in the release version that is being used for the docker image. I have changed this back to false. If we find that this is causing any problems in the future, we can bring it back.